cmake/OpenCVModule.cmake

```diff
@@ -75,6 +75,12 @@ set(OPENCV_MODULES_DISABLED_AUTO  "" CACHE INTERNAL "List of OpenCV modules impl
 set(OPENCV_MODULES_DISABLED_FORCE "" CACHE INTERNAL "List of OpenCV modules which can not be build in current configuration")
 unset(OPENCV_WORLD_MODULES CACHE)
 
+# set CMAKE_INSTALL_NAME_DIR if CMAKE_INSTALL_PREFIX isn't default value of "/usr/local"
+if(UNIX AND NOT ${CMAKE_INSTALL_PREFIX} STREQUAL "/usr/local")
```

Comment:
Does it work with `AND NOT DEFINED CMAKE_INSTALL_NAME_DIR` additional condition (we should not forbid to configure settings via command-line)?